curp/src/server/cmd_board.rs

```diff
@@ -28,7 +27,7 @@ pub(super) struct CommandBoard<C: Command> {
     /// Whether the cmd has been speculatively executed
     pub(super) spec_executed: HashSet<ProposeId>,
     /// Whether the cmd needs after sync
-    pub(super) needs_as: HashMap<ProposeId, LogIndex>,
+    pub(super) needs_as: HashMap<ProposeId, usize>,
```

Comment:
In rust, we can't use `u64` directly as an index, and except for persistence, this index does not need to consider size, so I use `usize` directly, and cast this index to `u64` when persisting

Reply:
Maybe it's ok to use usize across 32bit and 64bit systems as long as we use u64 as the persistency format for log index. And I think usize is too simple to require a type alias. If we use usize here and `LogIndex` there, it will also be confusing.